internal/venice-common/src/main/java/com/linkedin/venice/pubsub/adapter/kafka/admin/ApacheKafkaAdminConfig.java

```diff
@@ -20,7 +20,7 @@ public class ApacheKafkaAdminConfig {
   private final String brokerAddress;
 
   public ApacheKafkaAdminConfig(VeniceProperties veniceProperties) {
-    this.brokerAddress = ApacheKafkaProducerConfig.getPubsubBrokerAddress(veniceProperties);
+    this.brokerAddress = veniceProperties.getString(ApacheKafkaProducerConfig.KAFKA_BOOTSTRAP_SERVERS);
```

Comment:
Why do we need this change? The function also checks for Kafka using SSL too

Reply:
This config creation will be triggered by server and controller. For server and controller, before we pass the properties we will have some logic override `ApacheKafkaProducerConfig.KAFKA_BOOTSTRAP_SERVERS` field. And this logic is a bit different between server and controller. And the original goal of these logics for adpaters is: only respect this `ApacheKafkaProducerConfig.KAFKA_BOOTSTRAP_SERVERS` field. With this change, we pass all the configs instead of creating a new one, so `ApacheKafkaProducerConfig.getPubsubBrokerAddress` does not fit this, then it will cause a lot troubles.